tests/qos/files/mellanox/qos_param_generator.py

```diff
@@ -91,8 +92,7 @@ def collect_qos_configurations(self):
                 occupancy_per_port /= 2
                 pkts_num_trig_pfc_shp.append(occupancy_per_port + xon + hysteresis)
                 occupancy_per_port /= 2
-                ingress_ports_list_shp.append(ing_port)
-                ing_port += 1
+                ingress_ports_list_shp.append(testPortIds[i])
```

Comment:
Need to fix the dst_port_id as well

Reply:
Not really.
`dst_port_id` is generated by `QosSaiBase`. We don't touch it here.

Reply:
We overwrite that with the one obtained from qos.yml
https://github.com/Azure/sonic-mgmt/blob/master/tests/qos/test_qos_sai.py#L191

Reply:
Fixed